pwnagotchi/ui/view.py

```diff
@@ -238,7 +238,7 @@ def wait(self, secs, sleeping=True):
         part = secs / 10.0
 
         for step in range(0, 10):
-            # if we weren't in a normal state before goin
+            # if we weren't in a normal state before going
```

Comment:
I'm assuming this is a typo and not slang, there doesn't appear to be much slang in this corpus (it appears to be en-GB, and I think this slang would be en-US-something).